nailgun/entities.py

```diff
@@ -1000,6 +1003,26 @@ def build_pxe_default(self, synchronous=True, **kwargs):
         response = client.get(self.path('build_pxe_default'), **kwargs)
         return _handle_response(response, self._server_config, synchronous)
 
+    def clone(self, synchronous=True, **kwargs):
+        """Helper to clone an existing provision template
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.post(self.path('clone'), **kwargs)
+        result = _handle_response(response, self._server_config, synchronous)
```

Comment:
In order to follow the other helper methods, this helper should return the response here and then the caller should get this information and parse as needed. By doing that we have more options from the outside in order to check proper response before going ahead and getting the cloned template.